recipes/pvxs/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.3.3" %}
+
+package:
+  name: pvxs
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/epics-base/pvxs/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 944d16687fbdd31f43c617824048891756ff9edd93d209df260b17d2adc0b446
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - epics-base
+    - libevent
+  run:
+    - libevent
+
+test:
+  commands:
+    - pvxget -V
+
+about:
+  home: https://github.com/mdavidsaver/pvxs
+  license: BSD-4-Clause-UC
+  license_file: LICENSE
+  summary: Library and a set of CLI utilities acting as PVAccess protocol client and/or server
+
```

Comment:
You could add `doc_url` or `doc_src_url` (or both) as https://epics-base.github.io/pvxs/ 🔧

Reply:
I'm adding `doc_url`, both pointing to the same link would be too repetitive.